tests/test_upload.py

```diff
@@ -12,6 +14,13 @@
 access_token = 'pk.{0}.test'.format(
     base64.b64encode(b'{"u":"testuser"}').decode('utf-8'))
 
+UUID1 = 4  # chosen by fair dice roll, guaranteed to be random
```

Comment:
`4` -> `00000000-0000-0000-0000-000000000000` to reflect the current format of `rqp` value ?